README.adoc

```diff
@@ -182,34 +215,40 @@ The [hotspot=build file=0]`<build/>` section gives details of the two plugins th
 to build this project.
 
 * The Maven plugin for generating a WAR file as one of the output files.
-* The Liberty Maven plugin, which gives details of the name, version, and file type of 
-Open Liberty runtime package which it will download from the public Maven repository.
+* The Liberty Maven plugin, which allows you to install applications into Open Liberty and manage the server instances.
 
-In the [hotspot=liberty-maven-plugin file=0]`liberty-maven-plugin` plugin section, the [hotspot=server-name file=0]`<serverName/>` 
-field defines the name of the Liberty server that Maven creates. The [hotspot=bootstrap-properties file=0]`<bootstrapProperties/>` 
-section provides the set of variables that the [hotspot=http-port file=2]`server.xml` references. The 
-[hotspot=strip-version file=0]`<stripVersion/>` field removes the version number from the name of the application 
-(by default the version number of the application (`1.0-SNAPSHOT`) is appended to the 
-application name). This means that the application URL remains the same even when the version 
-number changes.
+In the [hotspot=liberty-maven-plugin file=0]`liberty-maven-plugin` plugin section, you can add a
+[hotspot=configuration file=0]`<configuration/>` element to specify Open Liberty configuration details.
+For example, the
+[hotspot=serverName file=0]`<serverName/>` field defines the name of the Open Liberty server that Maven creates.
+You specified the value for [hotspot=serverName file=0]`<serverName/>` to be `guideServer`.
+If the [hotspot=serverName file=0]`<serverName/>` field is not provided, the default value is `defaultServer`.
```

Comment:
Change
"In the [hotspot=liberty-maven-plugin file=0]`liberty-maven-plugin` plugin section, you can add a [hotspot=configuration file=0]`<configuration/>` element to specify Open Liberty configuration details. For example, the [hotspot=serverName file=0]`<serverName/>` field defines the name of the Open Liberty server that Maven creates. You specified the value for [hotspot=serverName file=0]`<serverName/>` to be `guideServer`. If the [hotspot=serverName file=0]`<serverName/>` field is not provided, the default value is `defaultServer`."
To
"In the [hotspot=liberty-maven-plugin file=0]`liberty-maven-plugin` plug-in section, you can add a [hotspot=configuration file=0]`<configuration/>` element to specify Open Liberty configuration details. For example, the [hotspot=serverName file=0]`<serverName/>` field defines the name of the Open Liberty server that Maven creates. You specified `guideServer` as the value for [hotspot=serverName file=0]`<serverName/>`. If the [hotspot=serverName file=0]`<serverName/>` field is empty, the default value is `defaultServer`."